src/repository/storage/postgres/orm/sequelize/userList.ts

```diff
@@ -0,0 +1,83 @@
+import type User from '@domain/entities/user.js';
+import type { UserList } from '@domain/entities/userList.js';
+import type { Sequelize } from 'sequelize';
+import { DataTypes } from 'sequelize';
+import { UserModel } from '@repository/storage/postgres/orm/sequelize/user.js';
+import type Orm from '@repository/storage/postgres/orm/sequelize/index.js';
+
+/**
+ * Class representing a table storing Users
+ */
+export default class UserListSequelizeStorage {
+  /**
+   * User model in database
+   */
+  public model: typeof UserModel;
+
+  /**
+   * Database instance
+   */
+  private readonly database: Sequelize;
+
+  /**
+   * Table name
+   */
+  private readonly tableName = 'users';
+
+  /**
+   * Constructor for user storage
+   *
+   * @param ormInstance - ORM instance
+   */
+  constructor({ connection }: Orm) {
+    this.database = connection;
+
+    /**
+     * Initiate user model
+     */
+    this.model = UserModel.init({
```

Comment:
I'm not sure that we need to create this model twice. On model creation, Sequileze creates a table in database as far as I know

Reply:
In that case should I remove the `userList` file from _repository/storage/postgres/orm/sequelize_ and move the [getUserList](https://github.com/codex-team/notes.api/blob/6390fad8c6ff8ff6caf996fa5975b2eb5997836b/src/repository/storage/postgres/orm/sequelize/userList.ts#L77) method to the `user` file? Otherwise this method cannot be accessed - exporting models from other classes is impossible.

Or maybe I should create a `UserListModel` here and work with it?